go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/getsentry/raven-go v0.1.0 // indirect
 	github.com/go-ini/ini v1.21.1 // indirect
 	github.com/go-kit/kit v0.8.0
+	github.com/go-logfmt/logfmt v0.4.0
```

Comment:
missed go.sum?

Reply:
Doesn't seem to need it?  I ran `go mod tidy` and it didn't add it.  Not sure what I'm suppose to do really.

Reply:
You need to run both `go mod tidy` and `go mod vendor`.
https://github.com/prometheus/prometheus/blob/master/CONTRIBUTING.md#dependency-management